File: blog-post-2/cpp/Greater.cpp
#include <iostream>
using namespace std;

int main()
{

    int num1, num2, num3, gt;

    cout <<"Enter three number: ";
    cin >> num1;
    cin >> num2;
    cin >> num3;
    
    if (num1 > num2)
    {
        gt = num1;
    }
    else if (num2 > num3)
    {
        gt = num2;
    }
    else
    {
        gt = num3;
    }
    cout <<"Greater number: " << gt;
        
}

File: blog-post-2/kotlin/Greater.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var scanner = Scanner(System.`in`)
    var num1:Int;var num2:Int;var num3:Int;var gt:Int
    System.out.print("Enter three number: ")
    num1 = scanner.nextInt()
    num2 = scanner.nextInt()
    num3 = scanner.nextInt()
    
    gt = if(num1 > num2){
        num1
    }
    else if(num2 > num3){
        num2
    }
    else{
        num3
    }
    println("Greater number: "+gt)
    
}